### PR TITLE
Super basic v2 starter

### LINKED ISSUE #843: Update/clean up front end dependencies for v2
# Description

## Action Items

Likely to be removed:

- [x] bezier-easing
- [x] bulma
- [x] bulma-checkradio
- [x] bulma-switch
- [x] gh-pages
- [x] leaflet
- [x] leaflet-pixi-overlay
- [x] leaflet.markercluster
- [x] pixi.js
- [x] react-leaflet
- [x] react-leaflet-choropleth
- [x] react-leaflet-control
- [x] react-leaflet-easyprint
- [x] react-leaflet-heatmap-layer
- [x] react-leaflet-markercluster
- [x] react-leaflet-pixi-overlay

Likely to be added:

- [x] @material-ui/core
- [x] @material-ui/icons (maybe)
more tbd

### PR DESCRIPTION
**Pull request by adamkendis**
Fixes #843 

barebones v2 starter

- Moved v1-specific stuff into `v1/` directory for now.
- Basic Header (no routing), Footer, stub for Map.
- Added a few v2 deps, removed some v1 deps.
- Commented out analytics/tracking scripts.

Screen Shot 2020-11-12 at 3 38 48 PM

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

**Review by tan-nate — APPROVED** [review]
Looks great!

File: client/package.json [hunk 1]
```diff
@@ -3,13 +3,12 @@
   "version": "0.1.0",
   "homepage": "https://www.311-data.org/",
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
```
Comment (jmensch1) [inline client/package.json:7]: love this!

File: client/package.json [hunk 2]
```diff
@@ -3,13 +3,12 @@
...
     "@react-pdf/renderer": "^1.6.8",
     "axios": "^0.19.0",
     "babel-jest": "^24.9.0",
     "bezier-easing": "^2.1.0",
-    "bulma": "^0.8.0",
-    "bulma-checkradio": "^1.1.1",
-    "bulma-switch": "^2.0.0",
```
Comment (jmensch1) [inline client/package.json:12]: and really love this!

File: client/theme/theme.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const theme = createMuiTheme({
+  palette: {
+    type: 'dark',
+  },
```
Comment (jmensch1) [inline client/theme/theme.js:6]: yessss

**Review by jmensch1 — APPROVED** [review]

**Pull request merged**